test/test_quantization.py

```diff
@@ -644,15 +644,18 @@ def checkQuantized(model):
     " Quantized operations require FBGEMM. FBGEMM is only optimized for CPUs"
     " with instruction set support avx2 or newer.",
 )
-@unittest.skip("temoprarily disable the test - enable after known issues are fixed")
 class GraphModePostTrainingQuantTest(QuantizationTestCase):
     @_tmp_donotuse_dont_inline_everything
     def test_single_layer(self):
-        r"""Quantize SingleLayerLinearModel which has one Linear module, make sure it is swapped
-        to nnq.Linear which is the quantized version of the module
+        r"""Compare the result of quantizing single linear layer in
+        eager mode and graph mode
         """
         # eager mode
-        model_eager = quantize(AnnotatedSingleLayerLinearModel(), test_only_eval_fn,
+        annotated_linear_model = AnnotatedSingleLayerLinearModel()
+        linear_model = SingleLayerLinearModel()
+        linear_model.fc1.weight = torch.nn.Parameter(annotated_linear_model.fc1.module.weight.detach())
```

Comment:
This is not a part of eager mode API, right? We should separate graph mode from eager mode and also comment each extra necessary step we need to make (i.e. it seems that we only need these lines in graph mode - we need to explain why).

Reply:
this is just making sure eager mode and graph mode models share the same parameters